src/client/components/app/Navigation/NavSidebar.tsx

```diff
@@ -78,6 +89,14 @@ const RouterLink = styled(Link)<{ selected: boolean }>`
       color: ${props.theme.colors.primary};
       font-weight: 700;
     }
+    &:hover {
+        ${LinkIcon} {
+      fill: ${props.theme.colors.primary};
```

Comment:
bad indent

Reply:
my auto prettier told me to do it that way - how should it be?